File: 71A - Way Too Long Words/WTLW.java
import java.util.Scanner;

public class WTLW {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int max = sc.nextInt();
        String[] tempStrings = new String[max];

        for (int i = 0; i < max; i++)
            tempStrings[i] = sc.next();

        for (String string : tempStrings)
            if (string.length() <= 10)
                System.out.println(string);
            else
                System.out.println("" + string.charAt(0) + (string.length() - 2) + string.charAt(string.length() - 1));

        sc.close();
    }
}

File: 71A - Way Too Long Words/WTLW.c
#include <stdio.h>
#include <string.h>

int main()
{
    int max;

    scanf("%d", &max);

    char tempStrings[max][200];

    for (int i = 0; i < max; i++)
        scanf("%s", tempStrings[i]);

    for (int i = 0; i < max; i++)
    {
        (strlen(tempStrings[i]) <= 10) ? printf("%s\n", tempStrings[i])
                                       : printf("%c%d%c\n", tempStrings[i][0], strlen(tempStrings[i]) - 2, tempStrings[i][strlen(tempStrings[i]) - 1]);
    }

    return 0;
}